packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -102,6 +102,35 @@
 
 	/* Display */
 
+	&.mod-togglePassword {
```

Comment:
why `mod-togglePassword` and not just `mod-password` ?

Reply:
I was afraid that we would want to put the `mod-password` on all the input password.

Reply:
we dont want that ?